### ISSUE TITLE
No output in headset. (OpenXR)

### ISSUE THREAD

**Issue by jpdery** [retroluxfilm/reshade-vrtoolkit#22]
Hi,

I just tried this with Population: One where this was working perfectly. However, since the last update (I think), when I start the game, I don't have an output in my headset. The game is running, but I'm staying in the steam vr "space" environment. Any ideas what is causing this ? Population: One is made from Unity.

Thanks

**Comment by jpdery** [comment]
Thanks for replying so quickly. Yeah I just tried the most recent  with no luck :(

**Comment by retroluxfilm** [comment]
:( guess OpenXR support must be implemented to reshade then to make work. Not sure when that will happen as the focus is not VR for the main reshade dev. (As reshade VR is essential for me I might look into it when I find time. )

**Comment by xerno2000** [comment]
> I just updated the release with a preview of the latest reshade code which includes OpenXR support. Make sure to use the reshade setup to configure it for OpenXR

I tried this on dcs. The menu appears on desktop but the changes do not show on the headset

**Comment by retroluxfilm** [comment]
> > I just updated the release with a preview of the latest reshade code which includes OpenXR support. Make sure to use the reshade setup to configure it for OpenXR
> 
> I tried this on dcs. The menu appears on desktop but the changes do not show on the headset

I just tried DCS through SteamVR and it works for me with the regular DLL method when placed in the DCS bin folder as described here (https://vrtoolkit.retrolux.de/gamelist.html) If you use OpenXR you need to install reshade to DCS with the Reshade installer I also added on the last release and select OpenXR.

2D and VR use indepedent reshade configs (reshade.ini & reshadeVR.ini). If you see the reshade icon in SteamVR dashboard once you are in a VR scene it will be working (https://vrtoolkit.retrolux.de/configuration.html). In case you dont use SteamVR you can only change the settings through the ini file or in the preset itself. To test you can change the preset in the reshadeVR.ini to a stronger variant (++) to directly see the difference better.

**Comment by xerno2000** [comment]
> > > I just updated the release with a preview of the latest reshade code which includes OpenXR support. Make sure to use the reshade setup to configure it for OpenXR
> > 
> > 
> > I tried this on dcs. The menu appears on desktop but the changes do not show on the headset
> 
> I just tried DCS through SteamVR and it works for me with the regular DLL method when placed in the DCS bin folder as described here (https://vrtoolkit.retrolux.de/gamelist.html) If you use OpenXR you need to install reshade to DCS with the Reshade installer I also added on the last release and select OpenXR.
> 
> 2D and VR use indepedent reshade configs (reshade.ini & reshadeVR.ini). If you see the reshade icon in SteamVR dashboard once you are in a VR scene it will be working (https://vrtoolkit.retrolux.de/configuration.html). In case you dont use SteamVR you can only change the settings through the ini file or in the preset itself. To test you can change the preset in the reshadeVR.ini to a stronger variant (++) to directly see the difference better.

Ah thats probably the issue then I dont use steamvr.
Is it possible to make changes through steam and then boot directly into the game without steam and the settings will be saved?

**Comment by retroluxfilm** [comment]
Yes that would be possible and acutally a good Idea. With that you can set everything up to your liking and then use OpenXR for better performance for non SteamVR headsets
